source/slang/hlsl.meta.slang

```diff
@@ -20225,3 +20225,22 @@ struct ConstBufferPointer
         }
     }
 }
+
+__glsl_version(450)
+__glsl_extension(GL_ARB_shader_clock)
+__target_intrinsic(glsl, clock2x32ARB)
+[require(glsl_spirv, GL_ARB_shader_clock)]
+uint2 clock2x32ARB()
```

Comment:
It looks like there is also a `uint64_t clockARB()` function [if GL_ARB_gpu_shader_int64 is defined](https://registry.khronos.org/OpenGL/extensions/ARB/ARB_shader_clock.txt)

Reply:
We are deprecating `target_intrinsic`. Please remove the line.